zh/configure-a-tidb-cluster.md

```diff
@@ -161,6 +161,19 @@ TiDB Operator 支持为 PD、TiDB、TiKV 挂载多块 PV，可以用于不同用
 
 PD、TiKV、TiDB、TiFlash、TiCDC 及 Pump 支持配置 Pod 使用宿主机上的网络命名空间 [`HostNetwork`](https://kubernetes.io/docs/concepts/policy/pod-security-policy/#host-namespaces)。可通过配置 `spec.hostNetwork: true` 为所有受支持的组件开启，或通过为特定组件配置 `hostNetwork: true` 为单个或多个组件开启。
 
+### SecurityContext
```

Comment:
We also need to update the following docs:
https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/deploy-tidb-operator
https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/advanced-statefulset
https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/enable-admission-webhook
https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/deploy-tidb-dm
https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/monitor-a-tidb-cluster
https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/initialize-a-cluster
https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/backup-restore-overview

Reply:
Add a new doc about how to run as non-root user for all of containers in security